tests/nodeos_multiple_version_protocol_feature_test.py

```diff
@@ -97,7 +97,8 @@ def hasBlockBecomeIrr():
                           specificExtraNodeosArgs={
                              0:"--http-max-response-time-ms 990000",
                              1:"--http-max-response-time-ms 990000",
-                             2:"--http-max-response-time-ms 990000"},
+                             2:"--http-max-response-time-ms 990000",
+                             3:"--last-block-time-offset-us -200000"},
                           onlySetProds=True,
```

Comment:
Maybe add a comment that 1.7 does not have the default of 1.8+ of -200000, so this is needed to avoid 'drop late blocks'.